build/wix/Localization/mixxx_cs-CZ.wxl

```diff
@@ -50,4 +51,48 @@
   <String Id="FeatureManualDescription">Uživatelská příručka Mixxxu v podobě PDF, dostupná též na http://mixxx.org/manual/</String>
   <String Id="FeatureTranslationsTitle">Překlady</String>
   <String Id="FeatureTranslationsDescription">Zdrojové soubory, které Mixxx umožní provedení instalace Mixxxu v jiných jazycích, než je americká angličtina</String>
+
+  <!-- Bootstrapper -->
+  <String Id="Caption">[WixBundleName] Setup</String>
+  <String Id="Title">[WixBundleName]</String>
+  <String Id="ConfirmCancelMessage">Are you sure you want to cancel?</String>
+  <String Id="ExecuteUpgradeRelatedBundleMessage">Previous version</String>
+  <String Id="HelpHeader">Setup Help</String>
+  <String Id="HelpText">/install | /repair | /uninstall | /layout [directory] - installs, repairs, uninstalls or&#13;&#10;    creates a complete local copy of the bundle in directory.&#13;&#10;    Install is the default.&#13;&#10;&#13;&#10; /passive | /quiet -  displays minimal UI with no prompts or displays no UI&#13;&#10;    and no prompts. By default UI and all prompts are displayed.&#13;&#10;&#13;&#10; /norestart   - suppress any attempts to restart. By default UI will prompt before restart.&#13;&#10;&#13;&#10; /log log.txt - logs to a specific file. By default a log file is created in %TEMP%.</String>
+  <String Id="HelpCloseButton">&amp;Close</String>
+  <String Id="InstallHeader">Welcome</String>
+  <String Id="InstallMessage">Setup will install Mixxx on your computer.&#13;&#10;It will first install required third party libraries, then Mixxx itself.&#13;&#10;Click install to continue or Close to exit.</String>
```

Comment:
Capitalization:
``Click install to continue or Close to exit``
 ->
``Click Install to continue or Close to exit``